conf/app.routes

```diff
@@ -150,7 +150,7 @@ GET         /language/:lang                                               contro
 #Payment history
 GET         /payment-refund-history                                       controllers.PaymentHistoryController.show(origin: Option[String] ?= None)
 GET         /agents/payment-refund-history                                controllers.PaymentHistoryController.showAgent
-GET         /refund-status                                                controllers.PaymentHistoryController.refundStatus(isAgent: Boolean = false)
+GET         /refund-status                                                controllers.PaymentHistoryController.refundStatus
```

Comment:
Yes the code was written so it would give the internal error page if called as an agent. The view file which contains the link to this page only calls for an individual user as well so it didn't make sense to call it with an isAgent flag and instead just restrict the endpoint to Individuals only.